wdae/wdae/common_reports_api/views.py

```diff
@@ -113,15 +111,14 @@ def post(self, request):
         return response
 
 
-class FamiliesDataTagDownload(QueryBaseView):
-    def get(self, request):
-        study_id = request.GET.get("study_id")
+class FamiliesDataDownloadView(QueryBaseView):
+    def get(self, request, dataset_id):
         tags = request.GET.get("tags")
 
-        if study_id is None:
+        if dataset_id is None or dataset_id == "":
```

Comment:
I think: `if dataset:` is shorter and cleaner.

Reply:
fixed

Reply:
fixed